assets/js/modules/pagespeed-insights/datastore/report.js

```diff
@@ -89,9 +91,81 @@ const baseSelectors = {
 
 		return reports[ `${ strategy }::${ url }` ];
 	},
+
+	/**
+	 * Gets report audits for the given strategy and URL.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Object|undefined)} Report audits.
+	 */
+	getAudits: createRegistrySelector( ( select ) => ( state, url, strategy, withStackPacks = false ) => {
+		const report = select( STORE_NAME ).getReport( url, strategy );
+		if ( report === undefined ) {
+			return undefined;
+		}
+
+		const { lighthouseResult } = report || {};
+		const { audits } = lighthouseResult || {};
+		if ( ! audits ) {
+			return {};
+		}
+
+		if ( withStackPacks ) {
+			const filteredAudits = {};
+
+			Object.keys( audits ).forEach( ( auditID ) => {
+				const stackPacks = select( STORE_NAME ).getStackPackDescriptions( url, strategy, auditID );
+				if ( Array.isArray( stackPacks ) && stackPacks.length > 0 ) {
+					filteredAudits[ auditID ] = audits[ auditID ];
+				}
+			} );
+
+			return filteredAudits;
+		}
+
+		return audits;
+	} ),
+
+	/**
+	 * Gets stack pack descriptions for a sepcific report audit.
+	 *
+	 * @since n.e.x.t
+	 *
+	 * @param {Object} state Data store's state.
+	 * @return {(Array.<Object>|undefined)} Stack pack descriptions for an audit.
+	 */
+	getStackPackDescriptions: createRegistrySelector( ( select ) => ( state, url, strategy, auditID ) => {
```

Comment:
Maybe instead of this selector (or in addition to that selector if we decide it's worth keeping this one) we should introduce a `getStackPackDescription( url, strategy, auditID, stackPackID )` that returns only the description for that specific audit and stack pack, which is closer to what we need here, since we should only display the WordPress stack pack descriptions.

Reply:
Updated it to be `getStackPackDescription( url, strategy, auditID, stackPackID )`